event_project/README.rst

```diff
@@ -0,0 +1,55 @@
+.. image:: https://img.shields.io/badge/licence-AGPL--3-blue.svg
+    :alt: License: AGPL-3
+
+Module name
+===========
+
+This module allows you to create events with assigned a project.
+You can make project template and assign this template to new event,
+this, create and calculate the project tasks dates.
+And if the event date start modify, recalculate tasks dates.
+
+
+Installation
```

Comment:
Don't put this section as all are standard modules. The section is reserved when you have any dependency.